### DISCUSSION TITLE
Paladin keeps stuck in few areas

### DISCUSSION THREAD

**Discussion by fridayaction** [isid0re/SoloLeveling#252] [Q&A]
hi isidore,

first of all I wanted to thank you for your script. it's easy to set up and for the most part very good. great work :)

One thing I noticed with my hammerdin:
he often gets stuck in areas such as mausoleum, pit, the temples in act 3 etc. and cannot get any further because he runs against a wall, for example. this happens especially in the difficulty level hell.
a text like "skipping [enemy name] [number]" is output in the console.
if you then continue to control the bot manually with the mouse and walk into an area where it could continue on its own again, it will walk back to the same starting point and keep walking against the wall. and he keeps doing it for very a very long time :D
for this reason a lot of time is lost and that is why it has only increased by around 3 levels in the last 12 hours.
current level is 83. 

maybe there is some easy way around this problem?

**Comment by isid0re** [comment]
I've noticed this as well. Unfortunately, there isnt a simple fix. It has to do with how the rooms are made for pathing within an area. Kolbot was originally designed for characters with teleportation. Once I finish other problems, i will be looking more into it.

**Comment by fridayaction** [comment]
thank you for the quick response.
looking forward to it.
again great work ;)
